### DISCUSSION TITLE
Multiple tags during a release

### DISCUSSION THREAD

**Discussion by cociweb** [arendst/Tasmota#17958] [Q&A] [answered]
Hello,
I used to do some CD activities after a new release is available from this awesome firmware. (Maybe that's my fault, but) I watching the tags to generate my custom firmwares: as soon a new tag is released, my buildings are starting. Unfortunatelly, I see that from time to time my builds created two times: The root cause is, a temporal tag is created. Last time with the v12.4.0 it was '[35](https://github.com/arendst/Tasmota/actions/runs/4193810281/jobs/7271487954#step:7:9)' but later than it will be deleted, then github gives 404 error on it.

My question is that, is it really needs: is it working intentionally as it designed or it is a room for improvement? If the first, I need to adjust my build trigger to check for only the 'vXX.Y.Z pattern.

Thank you!

**Comment by barbudor** [comment]
It is always like that
I don't know the reason
So better check for a tag starting with 'v'

**Reply by cociweb** [reply]
Ok.
Anyway, in the meantime I've found the corresponding part in the workflow:
https://github.com/arendst/Tasmota/blob/development/.github/workflows/Tasmota_build_master.yml#L195-L202

But I did not found yet the deleting and retagging part. :)

**Comment by arendst** [comment, marked as answer]
I am the deleting and retagging part!

The automation just builds the release without knowing what version it becomes and without valid releasnotes. So I have to edit and manage this. So just trigger on a vx.x.x tag and you're set to go.

**Reply by cociweb** [reply]
:D
Ok great!
Thank you guys!
